.travis.yml

```diff
@@ -1,25 +1,31 @@
 language: php
+sudo: false
 
 php:
+    - 7.1
     - 7.0
     - 5.6
     - 5.5
     - 5.4
     - 5.3
-    - 5.3.3
     - hhvm
     - hhvm-nightly
 
 matrix:
+    fast_finish: true
     allow_failures:
         - php: hhvm
         - php: hhvm-nightly
 
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 before_install:
     - composer self-update
 
 before_script:
-    - travis_retry composer install --no-interaction --prefer-source
+    - travis_retry composer install --no-interaction
```

Comment:
`travis_retry composer install --no-interaction --prefer-dist` (not sure if `--no-interaction` is necessary)